vipps-psp-api.md

```diff
@@ -133,6 +133,123 @@ Note 93 is for when the makePayment request from Vipps contains the statuses CAN
 
 To provide a consistent end user experience it is important that Vipps is notified by changes to the payment status when it is captured, cancelled or refunded. Vipps also provides an endpoint to check the payment status.
 
+# PSD2 Compliance
+
+In order to be allow for delegated SCA through the PSD2 directive every transaction needs to be marked with Vipps's WalletId. Vipps's WalletId is A62.
```

Comment:
Is this WalletId not specific for any card issuer? And would they know how to mark the transaction?